gunicorn/http/wsgi.py

```diff
@@ -300,10 +299,11 @@ def default_headers(self):
         headers = [
             "HTTP/%s.%s %s\r\n" % (self.req.version[0],
                 self.req.version[1], self.status),
-            "Server: %s\r\n" % self.version,
             "Date: %s\r\n" % util.http_date(),
             "Connection: %s\r\n" % connection
         ]
+        if self.cfg and self.cfg.server_tokens:
+            headers.append("Server: %s\r\n" % self.cfg.server_tokens)
```

Comment:
Noting that the header order will probably change here, which I suppose is not significant.